paraview/xmls/MergeTreeAutoencoder.xml

```diff
@@ -0,0 +1,653 @@
+      number_of_elements="1"
+      default_values="1"
+      panel_visibility="advanced">
+        <Documentation>
+          Normalize the metric loss by dividing by the maximum distance.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="NumberOfInit"
+      command="SetNumberOfInit"
+      label="Number of init."
+      number_of_elements="1"
+      default_values="4"
+      panel_visibility="advanced">
+        <Documentation>
+          Number of initialization steps, the best one will be kept.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="EuclideanVectorsInit"
+      command="SetEuclideanVectorsInit"
+      label="Euclidean Vectors Init"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+        <Documentation>
+          Initialize with Euclidean distance instead of the Wasserstein one. It will be faster but less accurate.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="InitOriginPrimeStructByCopy"
+      command="SetInitOriginPrimeStructByCopy"
+      label="InitOriginPrimeStructByCopy"
+      number_of_elements="1"
+      default_values="1"
+      panel_visibility="advanced">
+        <Documentation>
+          Initialize the tree structure of origins of the output bases by copying the structure of the origin of the corresponding input basis.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="TrackingLossDecoding"
+      command="SetTrackingLossDecoding"
+      label="TrackingLossDecoding"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+        <Documentation>
+          Choose to use the tracking loss in the decoding layers.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="TrackingLossInitRandomness"
+      command="SetTrackingLossInitRandomness"
+      label="TrackingLossInitRandomness"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+        <Documentation>
+          When the tracking loss is used, the origin of each output basis is the strict copy of the one in the input basis. This parameter allows to add randomess in this copy.
+        </Documentation>
+        <DoubleRangeDomain name="range" min="0" max="1" />
+      </DoubleVectorProperty>
+      
+      <IntVectorProperty
+      name="NumberOfProjectionSteps"
+      command="SetNumberOfProjectionSteps"
+      label="Number Of Projection Steps"
+      number_of_elements="1"
+      default_values="2"
+      panel_visibility="advanced">
+        <Documentation>
+          Number of steps to compute the projection of each input on a basis.
```

Comment:
Please clarify.

Reply:
thanks for providing the reference of the term in the paper and the section where it is mentioned.

Reply:
Done in the next commit!